docs/getting-started-v2-beta.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page-with-side-nav
+title: Getting Started
+---
+# Getting Started
+
+De 'Bevraging Persoon' Web API is gespecificeerd in OpenAPI specifications (OAS).
+
+Wil je de API gebruiken? Dit kun je doen:
+
+1. Bekijk de [functionaliteit en specificaties](#functionaliteit-en-specificaties)
+2. [Implementeer de API Client](#implementeer-de-api-client)
+3. [Probeer en test de API](#probeer-en-test-de-api)
+
+## Functionaliteit en specificaties
+
+Je kunt een visuele representatie van de specificatie genereren met [Swagger UI]({{ site.baseurl }}/swagger-ui) of [Redoc]({{ site.baseurl }}/redoc).
+
+De (resolved) OAS3 kun je hier downloaden: [openapi.yaml](https://github.com/VNG-Realisatie{{ site.baseurl }}/blob/v2.0.0beta1/specificatie/genereervariant/openapi.yaml).
+
+De [functionele documentatie](./features) vind je in [features](./features).
```

Comment:
Dit verwijst ook naar de huidige features. Spreken die de v2.0.0 niet tegen of valt dat wel mee ?